<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_account_form</class>
 <widget class="QWidget" name="edit_account_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Provider</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="provider_entry"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_Lookup">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Lookup</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_22">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Name identifying the website or online service
this account authenticates with.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>question_mark_icon.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>User</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="label_entry">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_24">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolTip">
          <string>A label to identify this account
such as your username or email address for the service.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>question_mark_icon.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Secret Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="PasswordInput" name="secret_entry">
         <property name="minimumSize">
          <size>
           <width>301</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_26">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The alphanumeric code shared with you by the provider.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>question_mark_icon.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordInput</class>
   <extends>QLineEdit</extends>
   <header>common_dialog_funcs</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>provider_entry</tabstop>
  <tabstop>label_entry</tabstop>
  <tabstop>secret_entry</tabstop>
  <tabstop>btn_Save</tabstop>
  <tabstop>btn_Cancel</tabstop>
  <tabstop>btn_Lookup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
